ui/pages/build-status.js

```diff
@@ -88,6 +88,22 @@ const RepoStatusItem = ({ repo, organization = 'Mender', branch = 'master', cove
   </Stack>
 );
 
+const buildStatusColor = (status) => {
+    switch(status) {
+    case 'FAILED':
+        return 'error';
+        break;
+    case 'RUNNING':
+        return 'inprogress';
+        break;
+    case 'SUCCESS':
+        return 'success';
+        break;
+    default:
+        return '#FFA500'; // Orange
+    }
+}
```

Comment:
```suggestion
const buildStatusColorMap = {
    FAILED: 'error',
    RUNNING: 'inprogress',
    SUCCESS: 'success',
    default: 'warning'
}
```
and then use it with `buildStatusColorMap[latestNightly.status] || buildStatusColorMap.default` or `buildStatusColorMap[latestNightly.status] ?? buildStatusColorMap.default`

Reply:
Oh my, whole new world :heart_eyes_cat: